dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/suite/KafkaUtils.java

```diff
@@ -51,4 +61,56 @@ private static void copyFileToServer(File file, LibertyServer server) throws Exc
         Files.delete(tmpDest);
     }
 
+    public static void copyTrustStoreToTest(ExtendedKafkaContainer container) throws IOException {
+        File file = container.getKeystoreFile();
+        Path tmpDest = Paths.get(TRUSTSTORE_FILENAME);
+        Files.copy(file.toPath(), tmpDest, StandardCopyOption.REPLACE_EXISTING);
+    }
+
+    /**
+     * Attempt to delete all the topics in the provided container*
+     *
+     * Due to the async and seemingly unreliable nature of kafka topic deletion this operation is done on a
+     * best effort basis and may leave topics in a `Marked for deletion` state
+     *
+     * If a test is susceptible to ending up in this state which is typically due to have >1 partition for a topic
+     * then using unique topic names per repeat is the recommended workaround
+     *
+     * @param container
+     * @throws IOException
+     * @throws ExecutionException
+     * @throws InterruptedException
+     */
+    public static void deleteKafkaTopics(ExtendedKafkaContainer container) throws IOException, ExecutionException, InterruptedException {
```

Comment:
This method is a bit horrible. We don't have enough information in the container itself to create the admin client so we're taking a guess at which class it came from based on the listener scheme and fishing the relevant properties out of there which isn't going to be reliable if we were to add more configurations.

Reply:
Can move the credential values from the Sasl test class into ExtendedKafkaContainer so that if other tests in the future need authentication details then we can easily reuse them and its cleaner from the perspective of cleanup as it is now from a common source accessible to all our containers.

Would need to add more helpers for the various properties or make it easier to pull the various Jaas configs from the container if we want to start exposing more of the security details to any operation wanting to build a connection to kafka